src/pages/Main.js

```diff
@@ -13,16 +13,28 @@ export default function Main({ onHandleSubmit }) {
   const [arrivalAirport, setArrivalAirport] = useState("");
   const [departureLocation, setDepartureLocation] = useState();
   const [arrivalLocation, setArrivalLocation] = useState();
+  const [id, setId] = useState();
 
   function onSubmit(event) {
     event.preventDefault();
     const form = event.target;
-    const { id } = form.elements;
     const { flightDate } = form.elements;
     const { flightTime } = form.elements;
     const { numberOfPassengers } = form.elements;
+    if (departureRunway === undefined && "--") {
+      alert("please select departure runway");
+      return null;
+    }
+    if (arrivalRunway === undefined && "--") {
+      alert("please select arrival runway");
+      return null;
+    }
+    if (aircraft === undefined && "--") {
```

Comment:
Should it not be: undefind || "--" ?

Reply:
thanks for the comment, alterations done